tests/jvm/src/test/resources/metac-metacp.expect.diff

```diff
@@ -39,16 +39,20 @@ _root_.com.javacp.MetacJava#
 ============================
 --- metac
 +++ metacp
-         symbol: "_root_.java.lang.Object#"
--    declarations: "_root_.com.javacp.MetacJava#`<init>`()."
--    declarations: "_root_.com.javacp.MetacJava#Overload1."
+     declarations: "_root_.com.javacp.MetacJava#`<init>`()."
+-    declarations: "_root_.com.javacp.MetacJava#Overload1#"
 -    declarations: "_root_.com.javacp.MetacJava#Overload1#"
--    declarations: "_root_.com.javacp.MetacJava#Overload2."
 -    declarations: "_root_.com.javacp.MetacJava#Overload2#"
-+    declarations: "_root_.com.javacp.MetacJava#<init>()."
```

Comment:
![](https://i.imgflip.com/260ng3.jpg)

Reply:
I chickened out of reviewing this diff. 🐔